R/utils.R

```diff
@@ -192,10 +193,23 @@ render_dependencies <- function(dependencies, local = TRUE, prefix=NULL) {
                                                                        dep[["stylesheet"]], 
                                                                        sep="/"))
     } else if ("stylesheet" %in% names(dep) & src == "file") {
+      dep[["stylesheet"]] <- paste0(path_prefix,
+                                "_dash-component-suites/",
+                                dep$name,
+                                "/",
+                                basename(dep[["stylesheet"]]),
+                                sprintf("?v=%s&m=%s", dep$version, modified))
```

Comment:
🐄 seems funny to combine `sprintf` and `paste0` - I guess that's because `modified` isn't a string, and I see this is used above too... but it would be easier to read if we pick one for any given concatenation.

Reply:
🙈fixed in d9d66c13062da253b0f77bb131126f64f12e8516